buildSrc/src/main/java/io/opentelemetry/instrumentation/gradle/muzzle/MuzzlePlugin.java

```diff
@@ -212,7 +212,7 @@ private static TaskProvider addMuzzleTask(
       Artifact versionArtifact,
       Project instrumentationProject,
       TaskProvider<?> runAfter) {
-    final String taskName;
```

Comment:
Still recommend `final` when variable isn't initialized inline

Reply:
Applies to many of the changes in this PR :P

Reply:
oh yes, reverted them all, though I'm not sure I understand the purpose for marking these vars final?

Reply:
When initialization happens deep in some control structure, accidentally double-initializing is a relatively common bug (or at least that's what I was taught when learning Java a long time ago :P)